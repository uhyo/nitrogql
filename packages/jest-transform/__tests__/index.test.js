import * as query from "./query.graphql";

it("snapshot test", () => {
  expect(query).toMatchSnapshot();
});
