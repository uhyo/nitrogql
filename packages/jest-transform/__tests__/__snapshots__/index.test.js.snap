// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
{
  "F1Fragment": {
    "definitions": [
      {
        "directives": [],
        "kind": "FragmentDefinition",
        "name": {
          "kind": "Name",
          "value": "F1",
        },
        "selectionSet": {
          "kind": "SelectionSet",
          "selections": [
            {
              "arguments": [],
              "directives": [],
              "kind": "Field",
              "name": {
                "kind": "Name",
                "value": "something2",
              },
            },
          ],
        },
        "typeCondition": {
          "kind": "NamedType",
          "name": {
            "kind": "Name",
            "value": "Query",
          },
        },
      },
    ],
    "kind": "Document",
  },
  "default": {
    "definitions": [
      {
        "directives": [],
        "kind": "OperationDefinition",
        "name": {
          "kind": "Name",
          "value": "Foo",
        },
        "operation": "query",
        "selectionSet": {
          "kind": "SelectionSet",
          "selections": [
            {
              "arguments": [],
              "directives": [],
              "kind": "Field",
              "name": {
                "kind": "Name",
                "value": "something",
              },
            },
            {
              "directives": [],
              "kind": "FragmentSpread",
              "name": {
                "kind": "Name",
                "value": "F1",
              },
            },
          ],
        },
        "variableDefinitions": [],
      },
      {
        "directives": [],
        "kind": "FragmentDefinition",
        "name": {
          "kind": "Name",
          "value": "F1",
        },
        "selectionSet": {
          "kind": "SelectionSet",
          "selections": [
            {
              "arguments": [],
              "directives": [],
              "kind": "Field",
              "name": {
                "kind": "Name",
                "value": "something2",
              },
            },
          ],
        },
        "typeCondition": {
          "kind": "NamedType",
          "name": {
            "kind": "Name",
            "value": "Query",
          },
        },
      },
    ],
    "kind": "Document",
  },
}
`;
